taskmanager/src/Domain/Ports/Incoming/AddTask.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace App\Domain\Ports\Incoming;
+
+interface AddTask
```

Comment:
IMHO here you put useless layer of the Ports&Adapters
I mean that actions like `AddTask` and `AssignTask` are more like "use cases". It means that such use cases should use adapters that implements ports to handle some actions`

e.g. 
`CLI/Controller/REST -> Request -> DTO -> Use Case (Application Layer) -> Port -> Adapter `

IMHO you could remove this incoming adapter due to unnecessary complications but I'm open for any discussions and arguments

Reply:
What do you understand by `use cases`? Can you explain? For me commandHandler can do the same.